res/http-export/configuration.toml

```diff
@@ -122,8 +122,11 @@ RetryWaitPeriod = "1s"
 Type="edgex-messagebus"
 SubscribeTopics="edgex/events/#"
   [Trigger.EdgexMessageBus]
-  Type = "redisstreams"
+  Type = "redis"
     [Trigger.EdgexMessageBus.SubscribeHost]
     Host = "localhost"
     Port = 6379
     Protocol = "redis"
+    [Trigger.EdgexMessageBus.Optional]
+    authmode = 'usernamepassword'  # required for redis messagebus (secure or insecure).
+    secretname = 'redisdb'
```

Comment:
any particular reason why the first letter of keys in this category is lower case?